safe_control_gym/experiments/base_experiment.py

```diff
@@ -174,6 +174,12 @@ def _select_action(self, obs, info):
             if success:
                 action = self.env.normalize_action(certified_action)
 
+        if self.last_step_timestep is not None and self.env.GUI is True:
+            # Sleep to maintain real-time pacing
+            elapsed = time.time() - self.last_step_timestep
+            time.sleep(max(0, 1.0 / self.env.CTRL_FREQ - elapsed))
```

Comment:
I'm wondering if we want a parameter to specify the sleep time instead of assuming "realtime" is favourable. I think the goal is to be able to visualize the system when the controller is too fast? It could have the default of being realtime but I think a parameter would be nice?

Reply:
Done!